docs/src/css/custom.css

```diff
@@ -38,8 +38,8 @@
 
   .menu__link--sublist {
     margin-top: 10px;
-    padding-bottom: 25px;
-    padding-top: 25px;
+    padding-bottom: 6px;
+    padding-top: 6px;
```

Comment:
If these are the Docusaurus defaults, could we simply remove these lines?

Reply:
Actually, this is a great idea. I have checked it too and it is working fine.

Reply:
I think we can remove.